README.md

```diff
@@ -11,7 +11,7 @@
 
 ---
 
-__Note: There is currently a lot of heavy experimenting going on in this repository, so it is not stable. Some of the information is not updated. Once the repository contents starts to stabilize, I will try to update the information texts.__
+__Note: There is currently a lot of heavy experimenting going on in this repository, so it is not stable. Some of the information is not updated. Once the repository contents starts to stabilise, I will try to update the information texts.__

```

Comment:
This isn't really a typo. It's just a difference in spelling between American and British English.